bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/FindReplaceLogicMessage.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Vector Informatik GmbH and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Vector Informatik GmbH - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.ui.internal.findandreplace;
+
+/**
+ * Avoid using this class. It is used as a glue to correctly map to the
+ * error-messages generated by RegEx-Errors which are directly displayed in
+ * plain text.
+ *
+ * Prefer using and extending {@link FindReplaceLogicStatus}
+ */
+public class FindReplaceLogicMessage implements IFindReplaceLogicStatus {
```

Comment:
I have additionally made it so that this Class cannot be just "passed any string" but requires a RegEx-Error to be constructed.

https://github.com/eclipse-platform/eclipse.platform.ui/commit/b8bf9a7e291031b53cf6666060cfd477933e3126